src/internal/operations/backup_integration_test.go

```diff
@@ -448,7 +447,7 @@ func toDataLayerPath(
 	case path.OneDriveService:
 		p, err = pb.ToDataLayerOneDrivePath(tenant, user, isItem)
 	default:
-		err = errors.Errorf("unknown service %s", service.String())
+		err = clues.Wrap(clues.New(service.String()), "unknown service")
```

Comment:
maybe another time to use `Sprintf` to avoid unnecessesary `Wrap` calls?

Reply:
sixes to me.  It's a nested pair of funcs either way.